docs/core/deploying/publish.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Publish your Application
+description: Publish your Application
+author: jamshedd
+ms.author: jamshedd
+ms.date: 09/06/2019
+ms.custom: 
+---
+# Publish your Application
+
+## Framework-Dependent Deployment and Self-Contained Deployment
+
+
+You can create various types of deployments for .NET Core applications:
+
+- **Framework-Dependent Deployment (FDD)**  
+  As the name implies, framework-dependent deployment (FDD) relies on the presence of a shared system-wide version of .NET Core on the target system. Because .NET Core is already present, your app is also portable between installations of .NET Core. Your app contains only its own code and any third-party dependencies that are outside of the .NET Core libraries. FDDs contain .dll files that can be launched by using the dotnet utility from the command line. For example, `dotnet app.dll` runs an application named app.
```

Comment:
I think we should be mentioning that with 3.0, FDDs are now producing executables so that you no longer have to launch via `dotnet app.dll`.

Reply:
Ack.